packages/zeebe-element-templates-json-schema/src/defs/icon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "#/icon",
+  "type": "object",
+  "description": "Custom icon to be shown on the element",
+  "default": {},
+  "properties": {
+    "contents": {
```

Comment:
Given our direction we could rename this as `source`. But of course `contents` would do, too.